pkg/cloudprovider/providers/aws/device_allocator.go

```diff
@@ -50,10 +50,15 @@ type deviceAllocator struct {
 // Allocates device names according to scheme ba..bz, ca..cz
 // it moves along the ring and always picks next device until
 // device list is exhausted.
+// Patch: now includes 'aa' all the way to 'dx'
 func NewDeviceAllocator(lastIndex int) DeviceAllocator {
 	possibleDevices := []mountDevice{}
-	for _, firstChar := range []rune{'b', 'c'} {
+	for _, firstChar := range []rune{'a', 'b', 'c', 'd'} {
 		for i := 'a'; i <= 'z'; i++ {
+			if firstChar == 'd' && i > 'x' {
+				continue; // skip 'y,z' for the 'd' range
```

Comment:
The thinking was breaking would be convenient since it's at the end of the check, but I am thinking of someone modifying this with some other checks that would interrupt it prematurely. It's a tradeoff between microoptimizing and introduce bugs along the way.